DumDum/TestProject1/DumDumServiceInMemoryDBTests.cs

```diff
@@ -0,0 +1,59 @@
+using DumDum.Controllers;
+using DumDum.Interfaces;
+using DumDum.Models;
+using DumDum.Models.JsonEntities;
+using DumDum.Models.JsonEntities.Kingdom;
+using DumDum.Models.JsonEntities.Player;
+using DumDum.Services;
+using Microsoft.Extensions.Options;
+using Moq;
+using Xunit;
+
+namespace TestProject1
+{
+    public class DumDumServiceInMemoryDBTests : TestService
+    {
+        private IAuthenticateService IAuthenticateService { get; set; }
+      
+        [Fact]
+        public void GetPlayerById_ReturnsPlayer_WhenPlayerExists()
+        {
+            //arrange
+            var UnitOfWork = GetContextWithoutData();
+            var DumDumService = new DumDumService(IAuthenticateService, UnitOfWork);
+            var testPlayer = DumDumService.
+                RegisterPlayerLogic(new PlayerRequest { KingdomName = "testk", Password = "123456789", Username = "user" }, out int StatusCode);
+            UnitOfWork.Kingdoms.Add(DumDumService.GetKingdomById(testPlayer.KingdomId));
```

Comment:
tady to saha do realne databaze nebo ne?

Reply:
ne, pouziva to ten inMemoryDB UnitOfWork
var UnitOfWork = GetContextWithoutData();